### PR TITLE
fix(distribution): refetch inquiry after editing for dialog preview

### PR DESCRIPTION
**Pull request by luytena**

### CODE REVIEW

File: packages/distribution/addon/gql/queries/inquiry-edit.graphql [hunk 1]
```diff
@@ -12,6 +14,7 @@ query InquiryEdit($inquiry: ID!) {
         }
         document {
           id
+          ...InquiryRequest
```
Comment (anehx) [inline packages/distribution/addon/gql/queries/inquiry-edit.graphql:17]: Why do we need the request in this query?

File: packages/distribution/addon/components/cd-inquiry-edit-form.js [hunk 2]
```diff
@@ -51,4 +55,12 @@ export default class CdInquiryEditFormComponent extends Component {
       );
     }
   }
+
+  async willDestroy() {
+    super.willDestroy();
+
+    await this.apollo.apolloClient.refetchQueries({
+      include: [inquiryEditQuery],
```
Comment (anehx) [inline packages/distribution/addon/components/cd-inquiry-edit-form.js:63]: Shouldn't we refetch the dialog query?

**Review by anehx — CHANGES_REQUESTED** [review]

File: packages/distribution/addon/components/cd-inquiry-edit-form.js (hunk 2 above)
Comment (luytena) [reply packages/distribution/addon/components/cd-inquiry-edit-form.js:63]: That works too, but then I refetch all of the inquiries on the dialog, when I only want the one inquiry. In addition, the inquiryDialogQuery seems to usually have multiple active observable queries, which then refetches all of them. Though I can ensure that only the dialog query of the current addressed / controlling group is fetched through the use of  onQueryUpdated.

Comment (anehx) [reply packages/distribution/addon/components/cd-inquiry-edit-form.js:63]: Let's just fire a query that fetches exactly what we need.. We don't really need to use "refetching".

Comment (luytena) [reply packages/distribution/addon/components/cd-inquiry-edit-form.js:63]: Ok, I'll rewrite it :+1:

**Pull request closed without merging**